specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/managedClusters.json

```diff
@@ -1280,6 +1280,101 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/privateLinkResources": {
+      "get": {
+        "tags": [
+          "privateLinkResources"
+        ],
+        "operationId": "PrivateLinkResources_List",
+        "summary": "Gets a list of private link resources in the specified managed cluster.",
+        "description": "Gets a list of private link resources in the specified managed cluster. The operation returns properties of each private link resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkResourcesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Private Link Resources by Managed Cluster": {
+            "$ref": "./examples/PrivateLinkResourcesList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/resolvePrivateLinkServiceId": {
```

Comment:
seems this operation is specific to aks. It's little confusing of "resolve private link service Id", from return type it should be "private link resource"?

Reply:
None of the stuff here is new. This is a merge from the dev branch to master. Nothing here is new.

Reply:
for "resolve private link service Id", it returns the private link resource as expected. BTW, privatelinkserviceId is part of private link resource.